src/components/AddMovie/AddMovie.css

```diff
@@ -0,0 +1,16 @@
+.button--transparent {
+	background: rgba(96, 96, 96, 0.68);
+	border-color: transparent;
+	border-radius: 4px;
+	width: 177px;
```

Comment:
177px is a strange number for a width. Usually to make ui look visually balanced dimensions of all ui elements should be divisible by some constant number.

Reply:
At least it is better to have all dimensions divisible by 2

Reply:
I used the values from the figma project requirements: https://www.figma.com/file/fKGjrOqR6nJe6LYJopGCZ8/%5BCDP%5D-Home-Task-%E2%80%93-React-v1?node-id=0%3A2